src/recipes/edge_search.js

```diff
@@ -0,0 +1,16 @@
+/* 
+    RECIPE: <NAME OF RECIPE>
```

Comment:
Just saw in a commit that you said this was removed. In the latest commit - it's still there.